benchmarks/benchmarks/stats.py

```diff
@@ -56,7 +56,97 @@ def time_ttest_ind_diff_var(self):
         stats.ttest_ind(self.a, self.c, equal_var=False)
 
 
+class DistributionsAll(Benchmark):
+    # all distributions are in this list. A conversion to a set is used to
+    # remove duplicates that appear twince in either `distcont` or
```

Comment:
```suggestion
    # remove duplicates that appear more than once in either `distcont` or
```
"more than once" is more familiar to me than "twince" : )